org.mixedrealitytoolkit.input/InteractionModes/INearInteractionModeTarget.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// This interface is used to update the enable state of the Components that are in the ProximityEnabledComponents
+    /// array (set in Editor) and to keep track of which Collider + XRBaseInteractor duples are triggering proximity.
+    /// The interface is needed to prevent a circular reference between MRTK Input and MRTK UX Core Scripts packages.
```

Comment:
Thank you for the nice comments.

Please put this second sentence in a <remarks> section, and add <see/> tags on types.  To improve readability of the learn.microsoft.com docs

```cs
    /// <summary>
    /// This interface is used to update the enable state of the components that are in the <see cref="ProximityEnabledComponents"/>
    /// array (set in Editor) and to keep track of which <see cref="Collider"/> and <see cref="XRBaseInteractor"/> duples are triggering proximity.
    /// </summary>
    /// <remarks>
    /// The interface is needed to prevent a circular reference between MRTK Input and MRTK UX Core Scripts packages.
    /// </remarks>
```

Reply:
Updated in last commit.  Resolving.